dependencies.py

```diff
@@ -295,22 +398,31 @@ def _install_dependencies(dependencies, target_dir, use_pypi=False):
         "--target",
         str(target_dir),
     ]
-    if use_pypi:
-        index = base_command.index("--find-links")
-        base_command.pop(index)  # --find-links
-        base_command.pop(index)  # the dir
-        base_command.pop(index)  # --no-index
 
-    for dependency in dependencies:
+    dialog = None
+    bar = None
+    startupinfo = None
+    if _is_windows():
+        dialog, bar = prog_dialog(0)
+        QApplication.processEvents()
+
+    for count, dependency in enumerate(dependencies):
         _uninstall_dependency(dependency)
         print("Installing '%s' into %s" % (dependency.name, target_dir))
         command = base_command + [dependency.name + dependency.constraint]
+
+        if _is_windows():
+            startupinfo = subprocess.STARTUPINFO()
+            startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW
+            dialog.setLabelText(f"Installing {dependency.name}")
```

Comment:
Twee keer "is_windows" en afhankelijk daarvan bestaat de dialog variabele die je de tweede keer aanroept. Het gaat goed, maar is wel gevaarlijk voor later refactoren.

Bij die tweede "if _is_windows" kan je beter "if dialog" doen, da's veiliger.